# Generated by Django 2.1.7 on 2019-11-04 18:57

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0031_info_trouble'),
    ]

    operations = [
        migrations.CreateModel(
            name='info_check_checker',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('log_id', models.CharField(max_length=64)),
                ('checker_name', models.CharField(max_length=32)),
                ('checker_id', models.CharField(max_length=32)),
            ],
        ),
        migrations.CreateModel(
            name='info_check_log',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('code', models.CharField(max_length=64)),
                ('enterprise', models.CharField(max_length=64)),
                ('address', models.CharField(max_length=128)),
                ('corporate_name', models.CharField(max_length=32)),
                ('corporate_job', models.CharField(max_length=32)),
                ('corporate_phone', models.CharField(max_length=16)),
                ('place', models.CharField(max_length=128)),
                ('begin_time', models.DateTimeField()),
                ('endtime', models.DateTimeField()),
                ('dept', models.CharField(max_length=32)),
            ],
        ),
    ]
